compare_mt.py

```diff
@@ -229,6 +231,38 @@ def print_ngram_report(ref, out1, out2,
     print('{}\t{} (sys1={}, sys2={})'.format(' '.join(k), v, match1[k], match2[k]))
   print()
 
+def print_word_reorder_error_report(ref, out1, out2,
```

Comment:
I think that once we create `RibesScorer` (see comment below), we can remove this function altogether.